src/submissions/components/SingleObservationForm.js

```diff
@@ -76,37 +76,48 @@ export default function SingleObservationForm() {
     setIsDeclinationOrElevationError
   ] = useState(false);
   // SUBMISSION UI STATES
-  const { jwt } = useAuthState(); // used in handleSubmit function
+  const { userAddress, authExpiry } = useAuthState(); // used in handleSubmit function
   const [showSubmitButton, setShowSubmitButton] = useState(false);
   const [isLoading, setIsLoading] = useState(false);
   // server provides success and error messages upon submissions which are displayed in UI
   const [successCount, setSuccessCount] = useState(null);
   const [errorMessages, setErrorMessages] = useState([]);
   // set to true if attempt to submit fails
   const [isError, setIsError] = useState(false);
+  const [
+    fetchObservationStationsError,
+    setFetchObservationStationsError
+  ] = useState(``);
+  const [
+    fetchObjectSearchResultsError,
+    setFetchObjectSearchResultsError
+  ] = useState(``);
 
   // gets observations stations for this user
   useEffect(() => {
     const fetchObservationStations = async () => {
       try {
         let result = await axios.post(
           `${API_ROOT}/getObservationStations`,
-          JSON.stringify({ jwt: jwt })
+          JSON.stringify({}),
+          {
+            withCredentials: true
+          }
         );
         // sort stations by most observations
         const sortedStations = result.data.sort(
           (a, b) => b.observation_count - a.observation_count
         );
         setObservationStations(sortedStations);
-      } catch (err) {
-        console.log(err);
+      } catch (error) {
+        setFetchObservationStationsError(error.response.data);
       }
     };
-    // only do fetch if user is logged in and the form does not already have their stations
-    if (jwt !== "none" && observationStations.length === 0) {
+    // only do fetch if form does not already have their stations
+    if (observationStations === ``) {
```

Comment:
Add catch to check if user is logged in before hitting the endpoint, i.e. `userAddress !== "none"`